### DISCUSSION TITLE
More liberal license?

### DISCUSSION THREAD

**Discussion by ceztko** [ttk592/spline#12] [General]
May I suggest a change to a more liberal license (MIT, LGPL, Apache2...)? I see it more appropriate for the scope and the amount of code. Of course it's your choice.

**Comment by ttk592** [comment]
I don't have a strong view regarding the license. Will review if other open source projects with incompatible license want to make use of it but will leave it for now. The intention of this project was more for educational purposes.

**Reply by mlyoung101** [reply]
Hi there,
I'm currently using this library in an unreleased project involving trajectory generation for a racecar, licenced under the MPL 2.0. I think it's a great library, but because of the GPL I would have to change my project's licence to the GPL in order to release it. Because my project also interacts with some other components, those might have to become GPL too, and the developers working on them might object to that happening.

All in all, a more permissive licence would be really helpful here so I can keep the MPL and release the project as-is.
